Source/WebCore/platform/graphics/ImageUtilities.h

```diff
@@ -51,6 +51,8 @@ enum class ImageDecodingError : uint8_t {
 };
 WEBCORE_EXPORT String descriptionString(ImageDecodingError);
 WEBCORE_EXPORT Expected<std::pair<String, Vector<IntSize>>, ImageDecodingError> utiAndAvailableSizesFromImageData(std::span<const uint8_t>);
+WEBCORE_EXPORT Vector<uint8_t> createIconFromImageData(std::span<const uint8_t> data, std::span<const unsigned> lengths);
```

Comment:
I think we might avoid this by using std::span in message; will try

Reply:
I believe you can construct a `FragmentedSharedBuffer` from your CFData/NSData and pass a the FragmentedSharedBuffer over IPC. Not only will it use span for encoding in the most case, it may also send shared memory if the CFData/NSData is mmap'd.

Reply:
ok; will update to `SharedBuffer`